charts/monasca/values.yaml

```diff
@@ -265,6 +331,20 @@ thresh:
   spout:
     metricSpoutThreads: 2
     metricSpoutTasks: 2
+  nimbus_resources:
+    requests:
+      memory: 512Mi
+      cpu: 100m
+    limits:
+      memory: 2Gi
+      cpu: 500m
+  supervisor_resources:
+    requests:
+      memory: 2Gi
```

Comment:
Will do, but it'll probably be best to wait until the new storm container is ready since storm's (excessively large) heap sizes are all hard-coded right now.